package.json

```diff
@@ -1,11 +1,17 @@
 {
   "name": "godaddy-test-tools",
-  "version": "12.0.2",
+  "version": "13.0.0",
   "description": "Gulp tools for simplifying the testing of node libraries with mocha and istanbul",
   "main": "index.js",
   "scripts": {
     "test": "gulp lint",
-    "postshrinkwrap": "gulp shrinkwrap --shrinkwrap.onlyFormat"
+    "postshrinkwrap": "gulp shrinkwrap --shrinkwrap.onlyFormat",
+    "prepush": "npm run test"
```

Comment:
alternative to git commit hooks huh? block push rather than commit?

Reply:
@jcrugzz Technically, that package creates a git push hook. It allows commits, but pushes, yes. I feel that is appropriate for a multi package repo that may have multiple commits on a single PR. (Like this one :))